addons/mail/views/mail_mail_views.xml

```diff
@@ -46,8 +46,6 @@
                                     <group string="Status">
                                         <field name="auto_delete"
                                             attrs="{'invisible': [('state', '!=', 'outgoing'), ('state', '!=', 'exception')]}"/>
-                                        <field name="to_delete"
```

Comment:
For stable, we could set invisible=1, and update the view in master, to lessen potential issues (does not cost much).

Reply:
yes, good idea :)